web/httpheader.jspf

```diff
@@ -66,6 +67,13 @@ org.opensolaris.opengrok.web.PageConfig"
     <link rel="stylesheet" type="text/css" href="<%=styleDir%>/jquery.combo.css%>" />
 --%>
 <%
+    cfg.addScript(ctxPath + "/js/jquery-3.2.0.min.js", 0);
```

Comment:
If I understand this correctly, the intention is to assemble a list of scripts and write them to the page in order according to priority. It seems that the ordering is known beforehand so instead of using the hand-tuned priority values, it might be better to have the list (sans versions) hardcoded in the correct order in the `Scripts` class and just record what scripts are necessary for given JSP (with `addScript()`) and them write them out in `getScripts()`)